docs/appcircle-api/faq.md

```diff
@@ -0,0 +1,224 @@
+            data={"pat": PERSONAL_ACCESS_TOKEN}
+        )
+        token_response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        print(f"Couldn't authenticate to {API_URL} with the PAT.")
+        print("Please check your PAT.")
+        print("If you are a self-hosted Appcircle user, please change the 'API_URL', 'AUTH_URL', and 'STORE_URL'")
+        sys.exit(1)
+
+    access_token = token_response.json().get("access_token")
+
+    print('Getting profiles...\n')
+    try:
+        profile_response = requests.get(
+            f"{API_URL}/store/v1/profiles",
+            headers={"authorization": f"Bearer {access_token}"}
+        )
+        profile_response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        print("Couldn't get the Enterprise App Store profiles.")
+        print("Please check your connection.")
+        sys.exit(1)
+
+    profiles = profile_response.json()
+    if not profiles:
+        print("No profiles found.")
+        sys.exit(1)
+
+    profile_id = profiles[0].get("id")
+    print(f"Enterprise App Store profile id: {profile_id}")
+
+    print("Getting app versions...\n")
+    try:
+        app_version_response = requests.get(
+            f"{API_URL}/store/v1/profiles/{profile_id}/app-versions?suborg=all",
+            headers={"authorization": f"Bearer {access_token}"}
+        )
+        app_version_response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        print("Couldn't get versions of the selected Enterprise App Store application.")
+        print("Please check your connection.")
+        sys.exit(1)
+
+    app_versions = app_version_response.json()
+    if not app_versions:
+        print("No app versions found.")
+        sys.exit(1)
+
+    app_version_id = app_versions[0].get("id")
+    app_os_type = app_versions[0].get("platformType")
+    app_os = "ios" if app_os_type == "1" else "android"
+    app_output_file = "app.plist" if app_os == "ios" else "app.apk"
+
+    print(f"App version id: {app_version_id}")
+    print(f"App OS: {app_os}")  # 1 for iOS, 2 for Android.
+    print("Getting app download link...\n")
+
+    print(f"Downloading the app to the {app_output_file} file.")
+    try:
+        download_response = requests.get(
+            f"{STORE_URL}/api/profile/{profile_id}/appversions/{app_version_id}/download-update",
+            headers={"Authorization": f"Bearer {access_token}"},
+            stream=True
+        )
+        download_response.raise_for_status()
+        with open(app_output_file, "wb") as f:
+            for chunk in download_response.iter_content(chunk_size=8192):
+                f.write(chunk)
+    except requests.exceptions.RequestException as e:
+        print(f"Couldn't download the app: {e}")
+        sys.exit(1)
+
+if __name__ == "__main__":
+    main()
+
+```
+
+  </p>
+</details>
+
+</TabItem>
+
+</Tabs>
+
```

Comment:
We could specify which variables should be changed in the script before using it.